package.json

```diff
@@ -9,8 +9,10 @@
 	"license": "Apache-2.0",
 	"private": true,
 	"scripts": {
-		"lint": "eslint packages --ext .ts",
-		"lint:fix": "eslint packages --ext .ts --fix",
+		"build": "rimraf dist && tsc -p tsconfig.base.json",
```

Comment:
Should just remove the `-p tsconfig.base.json`, rename the file to `tsconfig.json` and inside of the `tsconfig.eslint.json` change the `extend` to the normal `tsconfig.json` too